### PR TITLE
Add static information to guest sign in page

### PR DESCRIPTION
**Pull request by kyle1373**
The guest sign-in page originally contained just the title of the event. Because we will be transitioning to in-person events, I have added information on the guest sign-in page which will be displayed to the user whenever they sign in on a Chromebook, which are the date, time, event hosts, and event description. I have formatted this information to make it readable to users.

image

image

### CODE REVIEW

File: frontend/src/pages/EventSignInPage/index.tsx [hunk 1]
```diff
@@ -41,15 +42,41 @@ function EventSignInPage(): JSX.Element {
...
+                  >
+                    {format(parseISO(event.startDate), 'PP')} -{' '}
+                    {format(parseISO(event.startDate), 'p')} to{' '}
+                    {format(parseISO(event.endDate), 'p')}
+                  </Typography>
+                </Grid>
               </Grid>
 
+              <Grid item>
+                <Typography className={classes.eventName}
```
Comment (thaigillespie) [inline frontend/src/pages/EventSignInPage/index.tsx:67]: Need a `<Grid component>` within the `<Grid item>` here as well to cover the two `<Typography>`

**Review by thaigillespie — CHANGES_REQUESTED** [review]
Overall lgtm, there's only one little thing left and I'll approve after that's done

File: frontend/src/pages/EventSignInPage/index.tsx (hunk 1 above)
Comment (kyle1373) [reply frontend/src/pages/EventSignInPage/index.tsx]: Fixed

**Review by thaigillespie — APPROVED** [review]
lgtm

**Pull request merged**